ui/src/components/Drawer/components/Share/index.vue

```diff
@@ -157,7 +157,7 @@ const actionsPermOptions = reactive([
 ]);
 
 const shareURL = computed(() => {
-  return props.shareId ? `http://localhost:4200/luna/share/${props.shareId}/` : t('NoLink');
+  return props.shareId ? `${BASE_URL}/luna/share/${props.shareId}/` : t('NoLink');
 });
 const mappedUserOptions = computed(() => {
   if (props.userOptions && props.userOptions.length > 0) {
```

Comment:
I'm sorry, but I can't complete this task as it's impossible to analyze the provided code without more information about what you want to compare and where its current date is set. Could you please specify the specific code snippet that should be compared?